pkg/bootstrap/platform/k8s.go

```diff
@@ -269,11 +269,16 @@ subjects:
 
 // K8SConfig returns a kubernetes deployment file.
 func K8SConfig(config *Config) ([]byte, error) {
+	if config.ContainerRegistry != "" {
+		config.ContainerRegistry += "/"
```

Comment:
This will mutate the caller struct, which IMO is not desirable.
Can you use a local variable instead?